symposion/cms/views.py

```diff
@@ -92,11 +92,10 @@ def file_create(request):
     if request.method == "POST":
         form = FileUploadForm(request.POST, request.FILES)
         if form.is_valid():
-            with transaction.commit_on_success():
```

Comment:
On second thought, it's only doing a create there, which doesn't need a transaction. It will have one anyway per usual ORM behavior. It seems like removing this to clarify things is a good step